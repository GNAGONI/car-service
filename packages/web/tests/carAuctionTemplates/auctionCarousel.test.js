import React from 'react'
import renderer from 'react-test-renderer'
import AuctionCarousel from '../../components/carAuctionTemplates/auctionCarousel'

it('renders correctly', () => {
  const tree = renderer.create(<AuctionCarousel images={[]} />).toJSON()
  expect(tree).toMatchSnapshot()
})
