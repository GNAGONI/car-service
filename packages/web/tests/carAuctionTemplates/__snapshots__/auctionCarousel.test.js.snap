// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="banner-col"
>
  <div
    className="product-image-slider"
  >
    <div
      className="product-holder"
    >
      
      <div
        className="slick-slider slides slick-initialized"
      >
        
        <div
          className="slick-list"
        >
          <div
            className="slick-track"
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onMouseOver={[Function]}
            style={
              Object {
                "WebkitTransform": "translate3d(0px, 0px, 0px)",
                "WebkitTransition": "-webkit-transform 500ms ease",
                "msTransform": "translateX(0px)",
                "opacity": 1,
                "transform": "translate3d(0px, 0px, 0px)",
                "transition": "transform 500ms ease",
              }
            }
          />
        </div>
        
        
      </div>
    </div>
    <div
      className="slick-slider thumbnail-slider slick-initialized"
    >
      
      <div
        className="slick-list"
      >
        <div
          className="slick-track"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          onMouseOver={[Function]}
          style={
            Object {
              "WebkitTransform": "translate3d(0px, 0px, 0px)",
              "WebkitTransition": "-webkit-transform 500ms ease",
              "msTransform": "translateX(0px)",
              "opacity": 1,
              "transform": "translate3d(0px, 0px, 0px)",
              "transition": "transform 500ms ease",
            }
          }
        />
      </div>
      
      
    </div>
  </div>
</div>
`;
